extensions/amp-consent/0.1/test/test-consent-config.js

```diff
@@ -0,0 +1,244 @@
+      element.appendChild(script2);
+      expect(() => new ConsentConfig(element).getConfig())
+          .to.throw(multiScriptError);
+    });
+
+    it('remove not supported policy', () => {
+      toggleExperiment(win, 'multi-consent', false);
+      appendConfigScriptElement(doc, element, dict({
+        'consents': {
+          'ABC': 'r1',
+        },
+        'policy': {
+          'ABC': undefined,
+        },
+      }));
+      const consentConfig = new ConsentConfig(element);
+      expect(consentConfig.getConfig()['policy']).to.deep.equal({});
+    });
+  });
+
+  describe('expandPolicyConfig', () => {
+    it('create default policy', () => {
+      const policy = expandPolicyConfig(dict({}), defaultConfig['consents']);
+      expect(policy['default']).to.deep.equal({
+        'waitFor': {
+          'ABC': undefined,
+          'DEF': undefined,
+        },
+      });
+    });
+
+    it('create predefined _till_responded policy', function* () {
+      const policy = expandPolicyConfig(dict({}), defaultConfig['consents']);
+      expect(policy['_till_responded']).to.deep.equal({
+        'waitFor': {
+          'ABC': undefined,
+          'DEF': undefined,
+        },
+        'unblockOn': [
+          CONSENT_POLICY_STATE.UNKNOWN,
+          CONSENT_POLICY_STATE.SUFFICIENT,
+          CONSENT_POLICY_STATE.INSUFFICIENT,
+          CONSENT_POLICY_STATE.UNKNOWN_NOT_REQUIRED,
+        ],
+      });
+    });
+
+    it('create predefined _till_accepted policy', function* () {
+      const policy = expandPolicyConfig(dict({}), defaultConfig['consents']);
+      expect(policy['_till_accepted']).to.deep.equal({
+        'waitFor': {
+          'ABC': undefined,
+          'DEF': undefined,
+        },
+      });
+    });
+
+    it('create default _auto_reject policy', function* () {
+      const policy = expandPolicyConfig(dict({}), defaultConfig['consents']);
+      expect(policy['_auto_reject']).to.deep.equal({
+        'waitFor': {
+          'ABC': undefined,
+          'DEF': undefined,
+        },
+        'timeout': {
+          'seconds': 0,
+          'fallbackAction': 'reject',
+        },
+        'unblockOn': [
+          CONSENT_POLICY_STATE.UNKNOWN,
+          CONSENT_POLICY_STATE.SUFFICIENT,
+          CONSENT_POLICY_STATE.INSUFFICIENT,
+          CONSENT_POLICY_STATE.UNKNOWN_NOT_REQUIRED,
+        ],
+      });
+    });
+
+    it('override default policy', function* () {
+      const policy = expandPolicyConfig(dict({
+        'default': {
+          'waitFor': {
+            'ABC': [],
+          },
+        },
+      }), defaultConfig['consents']);
+      expect(policy['default']).to.deep.equal({
+        'waitFor': {
+          'ABC': [],
+        },
+      });
+      expect(policy['_till_accepted']).to.deep.equal({
+        'waitFor': {
+          'ABC': undefined,
+          'DEF': undefined,
+        },
+      });
+    });
+  });
+});
+
+function appendConfigScriptElement(doc, element, config) {
```

Comment:
nit: Can we move this to the top of the test? 😄

Reply:
yes, done